applications/StructuralMechanicsApplication/tests/test_truss_adjoint_element_3d2n.py

```diff
@@ -5,136 +5,125 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import math
 
-def get_displacement_vector(mp,disp):
-    index=0
-    for node in mp.Nodes:
-        disp[index] = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X,0)
-        index = index + 1
-        disp[index] = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y,0)
-        index = index + 1
-        disp[index] = node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Z,0)
-        index = index + 1
- 
-def add_variables(mp):
-    mp.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
-    mp.AddNodalSolutionStepVariable(KratosMultiphysics.REACTION)
-    mp.AddNodalSolutionStepVariable(KratosMultiphysics.VOLUME_ACCELERATION)
-    mp.AddNodalSolutionStepVariable(KratosMultiphysics.VELOCITY)
-    mp.AddNodalSolutionStepVariable(KratosMultiphysics.ACCELERATION)
-
-def apply_material_properties(mp,dim):
+def get_displacement_vector(model_part,disp):
```

Comment:
Personally i would prefer to create the `disp` vector inside this function and return it. Otherwhise you should check the size of the vector that is passed. (Even if this is just a local test function).

Reply:
Space after comma